zeyple/zeyple.py

```diff
@@ -12,6 +12,12 @@
 import gpgme
 import copy
 from io import BytesIO
+import dns.resolver
+myResolver = dns.resolver.Resolver()
+myResolver.nameservers = ['127.0.0.1', '8.8.8.8', '8.8.4.4']
+import urllib
+import gnupg
```

Comment:
We're already using gpgme. I'm not sure we need two different GPG library to do gpg. And, also, `gnupg` depending on your version can have very low security standards.